backend/backend/settings.py

```diff
@@ -126,6 +127,12 @@ def get_secret(secret_arn):
     },
     {
         "NAME": "authentication.password_validators.AlphaNumericPasswordValidator",
+    }
+    {
+        "NAME": "authentication.password_validators.PasswordStrengthValidator",
+        "OPTIONS" : {
+            "min_score": 3,
```

Comment:
interesting! I like the idea of having this here, but I've never worked with having to get this value. it looks doable (the [MinimumPasswordLength validator](https://github.com/django/django/blob/fe9c7ded2996364f853c524b4421274717d89d5f/django/contrib/auth/password_validation.py#L91) does it somehow) but i think this'll be hard and i don't want you to get stuck on something hard.

i'd suggest either 1) putting this value as a global constant at the top of the validator file (easiest) or 2) adding it in here at the top level as a value (so like `LANGUAGE_CODE` is below), which is not as easy but might be nicer? or might not. probably go with the easiest option